app/src/androidTest/java/com/github/se/polyfit/data/remote/firebase/PostFirebaseRepositoryFetchPosts.kt

```diff
@@ -0,0 +1,98 @@
+/*
```

Comment:
The tests that break other test on the CI.... Don't ask me why....

Reply:
Maybe something to do with asyc coroutines. 

Do you think this is something worth fixing in the future (making sure those tests pass)? If not, it might be a good idea to just delete this file to have a cleaner code base.

Reply:
Yes, but I'd like too keep them at least until it's linked with the Map, after I'll remove this evil tests